src/test/java/mocacong/server/domain/BaseTimeTest.java

```diff
@@ -0,0 +1,66 @@
+package mocacong.server.domain;
+
+import mocacong.server.domain.cafedetail.*;
+import mocacong.server.exception.notfound.NotFoundCafeException;
+import mocacong.server.repository.CafeRepository;
+import mocacong.server.repository.MemberRepository;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.test.autoconfigure.orm.jpa.DataJpaTest;
+
+import javax.persistence.EntityManager;
+import java.time.LocalDateTime;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+
+@DataJpaTest
+class BaseTimeTest {
+
+    @Autowired
+    MemberRepository memberRepository;
+    @Autowired
+    CafeRepository cafeRepository;
+    @Autowired
+    EntityManager entityManager;
```

Comment:
DataJpaTest를 사용하므로 TestEntityManager를 지원해줍니다.
테스트 전용 EntityManager를 가져와도 좋을 거 같아요
```suggestion
    TestEntityManager entityManager;
```